### DISCUSSION TITLE
How to set codeblock max content height and not scrolling?

### DISCUSSION THREAD

**Discussion by himself65** [fuma-nama/fumadocs#1018] [Q&A] [answered]
As title, I usually write some code sample in mdx, when content is getting long, the codeblock will overflow and there's scrollbar.

How to disable this behavior in mdx? not use react component

**Comment by fuma-nama** [comment, marked as answer]
We have a custom meta field allowed for code blocks

````md
```js custom="noscroll"
hello world
```
````

You can receive the `custom` prop from MDX components `pre` in page.tsx and turn it into a JSX `className`/`style` prop that specifies the CSS max height, it should be passed to `Pre` (see https://fumadocs.vercel.app/docs/ui/mdx/codeblock#usage).
